EIPS/eip-symbol.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: <to be assigned>
+title: Create `eth_symbol` method for JSON-RPC
+author: Peter Grassberger (@PeterTheOne)
+discussions-to: https://github.com/ethereum/EIPs/issues/3012
```

Comment:
Historically we have had one discussion issue per EIP, but I'm not opposed to having a single discussion issue for a bunch of related EIPs.  If other editors object we may need to break this out into multiple discussion issues though.